packages/cli/src/cli.ts

```diff
@@ -1,15 +1,15 @@
-import { build } from "gluegun";
+import {build, GluegunToolbox} from 'gluegun';
```

Comment:
Can we change the config to use `"` instead of `'`? Additionally, can we ensure all brackets have a space before and after values listed inside of them, for example: `{ build }` instead of `{build}`?

Both of these IMO help with readability, open to other opinions though.

Reply:
Yeah sure. Tbh I changed it because there was more instances of single quotes 😅